pyccl/ccl_sigM.i

```diff
@@ -0,0 +1,101 @@
+%module ccl_massfunc
```

Comment:
There is a bit too much going on in this interface file. Probably best to move most of the action to C.

Reply:
Done